backend/app/heartbeat_listener_sync.py

```diff
@@ -86,7 +86,6 @@ def callback(ch, method, properties, body):
                 new_feed = FeedDB(
                     name=extractor_name,
```

Comment:
You can also try add the description extractor_info["description"]. Something like this:
```
extractor_description = extractor_name
if "description" in extractor_info:
  extractor_description = extractor_info["description"]
```

Reply:
added